src/cmd_parser.c

```diff
@@ -305,6 +305,8 @@ check_and_set_compression_type(const char *type_str,
         *type = CR_CW_BZ2_COMPRESSION;
     } else if (!strcmp(compress_str->str, "xz")) {
         *type = CR_CW_XZ_COMPRESSION;
+    } else if (!strcmp(compress_str->str, "zstd") || !strcmp(compress_str->str, "zst")) {
```

Comment:
Why do you recognize non-standard "zstd" file name extension <https://datatracker.ietf.org/doc/html/rfc8878#section-7.1>? If there is no previous use of "zstd" extension in YUM repositories, I would not recognize it. It will only create unneeded pressure to other implementations (e.g. in DNF) to support both extensions "because createrepo_c" does that".

Reply:
+1

Reply:
That's a good suggestion, I have updated it, thanks.